### PR TITLE
Cleanup connection tests

### PR DESCRIPTION
**Pull request by genestack-solomatin**
Cleanup imports
Add consistency to string quotes
Add some missed tests
Rename test for free some space for tokens

### CODE REVIEW

File: test/test_connection.py [hunk 1]
```diff
@@ -28,23 +28,23 @@ def test_connection_to_wrong_url():
         connection.login(user_login, user_pwd)
 
 
-def test_login_positive():
+def test_login_positive_by_password():
```
Comment (klas-genestack) [inline test/test_connection.py]: `test_login_by_password_positive`?

File: test/test_connection.py [hunk 2]
```diff
@@ -28,23 +28,23 @@ def test_connection_to_wrong_url():
...
     connection = Connection(server_url)
     connection.login(user_login, user_pwd)
     name = connection.application('genestack/signin').invoke('whoami')
-    assert name == user_login, "Name does not match %s and  %s" % (name, user_login)
+    assert name == user_login, 'Name does not match %s and  %s' % (name, user_login)
 
 
-def test_open():
+def test_open_by_password():
```
Comment (klas-genestack) [inline test/test_connection.py]: `test_signin_by_password`?

File: test/test_connection.py [hunk 3]
```diff
@@ -53,6 +53,25 @@ def test_wrong_login():
         connection.login(user_pwd, user_pwd)
 
 
+def test_open_by_guest_negative():
```
Comment (klas-genestack) [inline test/test_connection.py]: `signin`?

File: test/test_connection.py [hunk 4]
```diff
@@ -53,6 +53,25 @@ def test_wrong_login():
...
+    connection = Connection(server_url)
+    connection.login(user_login, user_pwd)
+    name = connection.application('genestack/signin').invoke('whoami')
+    assert name == user_login
+
+
+def test_open_by_guest_positive():
```
Comment (klas-genestack) [inline test/test_connection.py]: Why here we don't use SignIn app?

File: test/test_connection.py [hunk 5]
```diff
@@ -53,6 +53,25 @@ def test_wrong_login():
...
+    connection = Connection(server_url)
+    connection.open('/')
+
+
+def test_guest_cannot_access_method_that_requre_login():
```
Comment (klas-genestack) [inline test/test_connection.py]: `test_method_forbidden_for_guest`

File: test/test_connection.py (hunk 4 above)
Comment (genestack-solomatin) [reply test/test_connection.py]: Looks like we should remove guest word and replace it with something else.

Comment (klas-genestack) [reply test/test_connection.py]: "unauthenticated"?

Comment (genestack-solomatin) [reply test/test_connection.py]: We don't have dedicated wrapper for `SignIn`.

File: test/test_connection.py [hunk 6]
```diff
@@ -28,23 +28,17 @@ def test_connection_to_wrong_url():
...
     connection.login(user_login, user_pwd)
     name = connection.application('genestack/signin').invoke('whoami')
-    assert name == user_login, "Name does not match %s and  %s" % (name, user_login)
-
-
-def test_open():
-    connection = Connection(server_url)
-    connection.login(user_login, user_pwd)
-    connection.open('/')
+    assert name == user_login, 'Name does not match %s and  %s' % (name, user_login)
```
Comment (klas-genestack) [inline test/test_connection.py]: `'Name ("%s") does not match login ("%s")'`

**Pull request merged**